jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java

```diff
@@ -158,14 +158,14 @@ public InputStream getBlobInputStream(ResultSet result, String columnName)
         if (_dbName == null)
             throw new IllegalStateException("DbAdaptor missing metadata");
 
-        if (_dbName.startsWith("postgres"))
-        {
+        try {
```

Comment:
The jetty style is to have open and close curlies on new lines. See https://www.eclipse.org/jetty/documentation/9.4.19.v20190610/coding-standards.html

Reply:
Fixed